cves/CVE-2013-4444.yml

```diff
@@ -4,13 +4,13 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname: Unrestricted File Upload Remote Code Execution
+nickname:
 
 CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 434-Unrestricted Upload of File with Dangerous Type
+CWE: 94
```

Comment:
I recommend taking a look at CWE 434 as it seems more appropriate in this situation